engine/src/main/java/org/hibernate/search/engine/cfg/spi/EngineSpiSettings.java

```diff
@@ -49,6 +50,17 @@ private EngineSpiSettings() {
 	 */
 	public static final String THREAD_PROVIDER = PREFIX + Radicals.THREAD_PROVIDER;
 
+	/**
+	 * The boolean flag that defines if Lucene executor service will accept nonblocking {@link OperationSubmitter#REJECTED_EXECUTION_EXCEPTION}
+	 * operation submitter. If set to {@code true} and Lucene executor service receives {@link OperationSubmitter#REJECTED_EXECUTION_EXCEPTION}
+	 * operation submitter then an exception will be thrown, otherwise a submit operation will proceed.
+	 * <p>
+	 * Expects a boolean {@code true/false}.
+	 * <p>
+	 * Defaults to {@link Defaults#IS_WRITE_EXECUTOR_BLOCKING}, {@code true}.
+	 */
+	public static final String IS_WRITE_EXECUTOR_BLOCKING = PREFIX + Radicals.IS_WRITE_EXECUTOR_BLOCKING;
```

Comment:
Please don't forget to remove changes to this file when your remove the configuration property.